### DISCUSSION TITLE
Something that cycles through every color

### DISCUSSION THREAD

**Discussion by loglot** [TodePond/Arroost#273] [Show and tell]
https://github.com/TodePond/Arroost/assets/88983354/95f82adc-bfb8-4df7-ab00-fd898c6b92ce

**Comment by loglot** [comment]
I got some spaghetti code

**Comment by mahdoosh1** [comment]
Wow.
Look how much time you've spent.

Try making something that switches wire color back and forth.

**Reply by loglot** [reply]
it was difficult because i was on my phone to make this, i just found out how to add colors together, so i wanted to make this

**Reply by loglot** [reply]
calculator next lol

**Reply by mahdoosh1** [reply]
Oh rip.
